src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,70 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] entries = getTableEntries(fromFileName);
+        if (entries.length == 0 || entries[0].isEmpty()) {
```

Comment:
The check for `entries.length == 0 || entries[0].isEmpty()` might not be sufficient to determine if the file is empty or contains only empty lines. Consider checking if all entries are empty or if the file itself is empty before proceeding.